### PR TITLE
Use correct encoding to produce hyphen-minus when rendering man pages

### PR DESCRIPTION
**Pull request by yrro**
When a user searches a man page in e.g, less, when they search for - less searches for U+002D HYPHEN MINUS.

As of groff 1.2, - renders out as U+2010 HYPHEN.

Therefore, it has become difficult to search for options in man pages. This change switches man pages to use the escape sequence \\- which renders out as U+002D HYPHEN MINUS.

See: groff_man(7), <https://lwn.net/Articles/947941/>

### CODE REVIEW

**Review by sysstat — COMMENTED** [review]
Thanks for this patch. Please take notice of my remark about using `\[mi]`.

File: man/sar.in [hunk 1]
```diff
@@ -1640,12 +1640,12 @@ SGR substring for restart messages.
 .BR "W=" " (or " "M=" ")"
 SGR substring for percentage values in the range from 75% to 90% (or in the range 10% to 25%
 depending on the metric's meaning).
-It is also used for negative values in the range from -10 to -5.
+It is also used for negative values in the range from \[mi]10 to \[mi]5.
```
[the remark this answers is not in the document]
Comment (yrro) [reply man/sar.in:1643]: Fair enough, I'll make that change too 👍

**Comment by yrro** [comment]
I've switched all `\[mi]` to `\-` now. Might be nice to have it for the `kbmemused` formula, up to you.

**Comment by sysstat** [comment]
Thanks. PR merged.

**Pull request merged**